tests/IndexBuilderTest.php

```diff
@@ -206,6 +206,38 @@ public function testPurgerDistinguishesIndicesWithTheSamePrefix(): void
         $this->assertCount(0, $operations);
     }
 
+    /** @dataProvider purgeIndexBuilderProvider */
+    public function testDoNotPurgeAndCloseOldIndicesInDryMode(IndexBuilder $indexBuilder): void
+    {
+        $index1 = $indexBuilder->createIndex('hop');
+        $this->assertInstanceOf(Index::class, $index1);
+
+        usleep(1200000); // 1,2 second
```

Comment:
I wonder why you need to sleep here?
If you need it because we use the current timestamps in the index name, you can use the symfony helper for that https://symfony.com/doc/current/components/phpunit_bridge.html#time-sensitive-tests

It allows to run test as fast as possible

Reply:
To be honest, I don't know why there is a need for a sleep here.
My test is largely, even entirely, inspired by @damienalexandre 's [testPurgeAndCloseOldIndices](https://github.com/jolicode/elastically/blob/master/tests/IndexBuilderTest.php#L144-L181) test.

However, I will update my test in order to use the symfony helper.

Reply:
I've added the `@group time-sensitive` annotation on my test.

Reply:
Wonderful :)